docs/user-guide/persistent-volumes/index.md

```diff
@@ -222,6 +222,51 @@ A volume will be in one of the following phases:
 
 The CLI will show the name of the PVC bound to the PV.
 
+### Mount Options
+
+Kubernetes administrator can specify additional mount option to be used when a Persistent Volume is
```

Comment:
I was thinking in some cases, the reader may not have access to create/view/edit the Persistent Volume.

So it could be a cluster admin or something like that.